staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -127,6 +127,10 @@ type ClientConfigLoadingRules struct {
 	// DefaultClientConfig is an optional field indicating what rules to use to calculate a default configuration.
 	// This should match the overrides passed in to ClientConfig loader.
 	DefaultClientConfig ClientConfig
+
+	//WarnIfAllMissing indicates whether the configuration files pointed by KUBECONFIG environment variable are present or not.
```

Comment:
gofmt might complain about the missing space after `//`

Reply:
It didn't. But I have Added space here.